doc/api/v8.md

```diff
@@ -134,3 +134,150 @@ setTimeout(function() { v8.setFlagsFromString('--notrace_gc'); }, 60e3);
 [V8]: https://developers.google.com/v8/
 [here]: https://github.com/thlorenz/v8-flags/blob/master/flags-0.11.md
 [`GetHeapSpaceStatistics`]: https://v8docs.nodesource.com/node-5.0/d5/dda/classv8_1_1_isolate.html#ac673576f24fdc7a33378f8f57e1d13a4
+
+## Serialization API
+
```

Comment:
If we are going to introduce this API, even tho I know this is going into the `v8` module that is already clearly marked as being fluid based on what v8 chooses to do, we should mark this explicitly as being Experimental for the time being.